security/gopass/Portfile

```diff
@@ -0,0 +1,370 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/gopasspw/gopass 1.10.1 v
+categories          security
+maintainers         {@sikmir gmail.com:sikmir} openmaintainer
+license             MIT
+
+description         The slightly more awesome Standard Unix Password Manager for Teams
+long_description    ${description}
```

Comment:
Hey @sikmir, this looks good.  2 minor nitpicks though:

Usually when reusing the `$description` variable in `long_description`, it's written as `{*}${description}` for proper expansion. ([Example](https://github.com/macports/macports-ports/blob/18a358df025b385effbab1677b74253aa5b86701/databases/mariadb-10.5/Portfile#L321))